packages/slice-machine/src/models/slice/context.tsx

```diff
@@ -30,7 +17,7 @@ export function useModelReducer({
   remoteSlice?: SliceSM;
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   mockConfig: any;
-}): [SliceState, SliceStore] {
+}): SliceState {
```

Comment:
I think this and those types 'SliceState' and LibraryState add an extra layer of complexity.
Hope that at some point we can get rid of it and do something a little simpler like the custom type stores.

Reply:
Yes tbh I have not fully understood why we have those, maybe we can add something related to it in the tech debt backlog?